### DISCUSSION TITLE
Can I load custom glsl shaders?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [kovidgoyal/kitty#6651] [Q&A]
Hello there and thanks for this outstanding terminal.

Problem is: my characters aren't shiny enough. I already applied opacity, figured out a nice shell, a palette, set a blur, found an amazing font with ligatures. But you see, it's day and even though there's light, characters in my terminal don't reflect it.

Of course, I cannot work in such an environment where my characters don't shine enough.

Soooo the question is. Can I run custom glsl shaders against kitty?

Thanks!

**Comment by SolitudeSF** [comment]
you can run kitty with zink, which enables you to use vkBasalt.
